recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/guest-addons.bb

```diff
@@ -40,18 +41,25 @@ FILES_${PN}-run-doma += " \
     ${base_prefix}${XT_DIR_ABS_ROOTFS_SCRIPTS}/start_doma.sh \
 "
 
+FILES_${PN}-run-domf += " \
+    ${sysconfdir}/init.d/guest_domf \
+"
+
 PACKAGES += " \
     ${PN}-run-domd \
     ${PN}-run-doma \
+    ${PN}-run-domf \
 "
 
 # configure init.d scripts
-INITSCRIPT_PACKAGES = "${PN}-run-domd ${PN}-run-doma"
+INITSCRIPT_PACKAGES = "${PN}-run-domd ${PN}-run-doma ${PN}-run-domf"
 
 INITSCRIPT_NAME_${PN}-run-domd = "guest_domd"
 INITSCRIPT_PARAMS_${PN}-run-domd = "defaults 85"
 INITSCRIPT_NAME_${PN}-run-doma = "guest_doma"
 INITSCRIPT_PARAMS_${PN}-run-doma = "defaults 86"
+INITSCRIPT_NAME_${PN}-run-domf = "guest_domf"
+INITSCRIPT_PARAMS_${PN}-run-domf = "defaults 85"
```

Comment:
I suggest to specify order number bigger than guest_domd.

Reply:
87 or 86?

Reply:
if 86, then android shift to 87

Reply:
Maybe 86 is better for domf. In principle telemetry is needed earlier than Android HMI.